src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
+
+  if (days >= 7) {
+    sum -= 50;
+  } else if (days >= 3) {
+    sum -= 20;
```

Comment:
move 30, 40 and 50 in the variables

Reply:
not fixed 
<img width="318" alt="image" src="https://user-images.githubusercontent.com/99331793/206414635-61bcf7c1-4a07-4227-b37e-df1ccae11c43.png">